src/components/FindMovie/FindMovie.tsx

```diff
@@ -16,42 +77,58 @@ export const FindMovie: React.FC = () => {
               type="text"
               id="movie-title"
               placeholder="Enter a title to search"
-              className="input is-danger"
+              value={query}
+              className={cn('input', {
+                'is-danger': onloadError,
+              })}
+              onChange={handleSearchChange}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {onloadError && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
```

Comment:
You render the same error in every case.
Let's change `useState` with error to string and set different error messages in different cases.